.github/workflows/lint_and_test.yaml

```
@@ -0,0 +1,35 @@
+name: Python package
+
+on: [push]
```

```suggestion
  push:
    branches: ["main"]
  pull_request:
    branches: ["main"]
```
I think we want this to be triggered also when new PRs are created.